spec/features/idv/doc_auth/document_capture_step_spec.rb

```diff
@@ -67,12 +67,10 @@
     end
 
     expect(page).to have_current_path(idv_session_errors_throttled_path)
-    # Bug: Rate limit event is not always logged (LG-6543)
-    # expect(fake_analytics).to have_logged_event(
-    #   'Throttler Rate Limit Triggered',
-    #   throttle_type: :idv_doc_auth,
-    # )
-    # expect(fake_attempts_tracker).to receive(:idv_document_upload_rate_limited)
```

Comment:
Can bring this back now? assuming this change fixed the commented out bug?
```suggestion
    expect(fake_attempts_tracker).to receive(:idv_document_upload_rate_limited)
```

Reply:
Not just yet, still dealing with errors in that test. Wanted CI to make sure where I was calling these wasn't borking other tests